dashboard/app/assets/stylesheets/application.scss

```diff
@@ -350,28 +350,74 @@ img.video_thumbnail {
     }
 
     .user_menu_arrow_down,
-    .user_menu_arrow_up {
+    .user_menu_arrow_up,
+    .create_menu_arrow_down,
+    .create_menu_arrow_up, {
```

Comment:
We could also create a common class for these, say `.create_menu_arrow`.

Reply:
I use the down and up versions for separate styling later in the file and these classes are consistent with user_menu.